src/System.Private.ServiceModel/src/System/ServiceModel/Channels/WebSocketTransportDuplexSessionChannel.cs

```diff
@@ -878,7 +873,7 @@ Message PrepareMessage(WebSocketReceiveResult result, byte[] buffer, int count)
                     Message message;
                     if (_useStreaming)
                     {
-                        using (var wrappedStream = new MaxMessageSizeStream(
+                        var wrappedStream = new MaxMessageSizeStream(
```

Comment:
Yes. Indirectly this stream ends up being contained inside another class who's name eludes me right now which itself implements Stream and is what gets returned to user code when an operation has a return type of Stream and using streamed transfer mode. The user code can then call Dispose() on that class and I presume that propagates the Dispose() call down the line.